Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:8
+
+RUN yarn global add kinesalite
+
+RUN groupadd kinesalite && useradd --gid kinesalite kinesalite
+
+RUN mkdir /var/kinesalite
+RUN chown kinesalite /var/kinesalite
+
+USER kinesalite
+
+EXPOSE 4567
+ENTRYPOINT ["kinesalite", "--path", "/var/kinesalite"]
```

Comment:
Do we need to specify path here?
If the primary purpose of the library is correctness testing I wouldn't have thought there would be much data passing through, and the keep-in-RAM mode would be faster.

Path can always be added at the run stage if people need it.

Reply:
No and fair point, removed.